server/plugin.go

```diff
@@ -46,8 +46,8 @@ const (
 	SettingReminders              = "reminders"
 	SettingOn                     = "on"
 	SettingOff                    = "off"
-
-	chimeraGitLabAppIdentifier = "plugin-gitlab"
+	ActionNameMemberAccessRequest = "member_access_requested"
```

Comment:
I think we should leave a blank line after chimera constant and then ActionName so future code dependency